spaceship/lib/spaceship/du/du_client.rb

```diff
@@ -45,6 +45,10 @@ def upload_trailer_preview(app_version, upload_file, content_provider_id, sso_to
       upload_file(app_version: app_version, upload_file: upload_file, path: '/upload/image', content_provider_id: content_provider_id, sso_token: sso_token_for_image, du_validation_rule_set: screenshot_picture_type(device, nil))
     end
 
+    def upload_app_review_attachment(app_version, upload_file, content_provider_id, sso_token_for_image)
```

Comment:
`sso_token_for_image` - should this be something like `sso_token_for_file` or `sso_token_for_attachment` instead?